src/Microsoft.SqlTools.ServiceLayer/Migration/MigrationService.cs

```diff
@@ -402,6 +406,75 @@ public void InitializeService(ServiceHost serviceHost)
             }
         }
 
+        internal async Task HandleSaveAssessmentRequest(
+            SaveAssessmentResultParams parameters,
+            RequestContext<SaveAssessmentResult> requestContext)
+        {
+            try
+            {
+                DmaEngine engine = new DmaEngine();
+                var assessmentReportFileName = String.Format("SqlAssessmentReport-{0}.json", DateTime.UtcNow.ToString("yyyyMMddHHmmss", CultureInfo.InvariantCulture));
+                engine.SaveAssessmentResultsToJson(parameters.AssessmentResult, false, Path.Combine(SqlAssessmentConfiguration.ReportsAndLogsRootFolderPath, assessmentReportFileName));
```

Comment:
the folder path should be passed by customer？ not from default folder path?

Reply:
for first iteration we are using default folder path. may be later we can give option to users to provide folder just for reports.